src/reanimated2/hook/commonTypes.ts

```diff
@@ -2,13 +2,17 @@ export type DependencyList = Array<unknown> | undefined;
 
 export type Context = Record<string, unknown>;
 
+export type ShadowNodeWrapper = object;
```

Comment:
Shouldn't this be in the other `commonTypes` file (the one under `reanimated2/`), since it's used in more places than hooks?

Reply:
Done in 78711e57f7afd58aac482e9a4609ea2afad8ce36